applications/ISO20022.md

```diff
@@ -0,0 +1,185 @@
+
+### Team Code Repos
+- [Order Food on the Blockchain](https://github.com/InkSmartContract/BlockchainFoodOrder)
+
+Please also provide the GitHub accounts of all team members. 
+- [GitHub1](https://github.com/OpenSmartContract) 
+- [Github2](https://github.com/FPLeader) 
+
+### Team LinkedIn Profiles (if available)
+- [Profile](https://www.linkedin.com/in/dengdaniel)
+
+## Development Status 
+
+- [RFPs](https://github.com/w3f/Grants-Program/tree/master/docs/RFPs) (requests for proposal)
+- [Blockchain Food Order dApp](https://github.com/InkSmartContract/BlockchainFoodOrder/blob/main/Learn%20ink!%20by%20Example%20%E2%80%94%20Order%20Food%20on%20the%20Blockchain.pdf)
+- [Smart Contract Deployment and Interaction Guide using Contracts-UI and Polkadot.js](https://github.com/InkSmartContract/BlockchainFoodOrder/blob/main/BlockchainFoodOrder%20Smart%20Contract%20Deployment%20and%20Interactions%20Guide.pdf)
+
+## Development Roadmap 
+
+### Overview
+
+- **Total Estimated Duration:** ~3-6 months
+- **Full-Time Equivalent (FTE):**  1,4 FTE
+- **Total Costs:** 3 BTC 
+
+### Milestone 1 Example — Basic functionality
+
+- **Estimated duration:** ~1-2 months
+- **FTE:**  1,2
+- **Costs:** 0.95 BTC
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | Design documents with details on what it takes for Polkadot/Substrate to support ISO20022 messages, gaps and solution recommendations |
+| 1. | Design the use case | Degine an ISO20022 cross border payment use case with persona, end-to-end flow | 
+| 2. | Specify ISO20022 messages | Specify ISO20022 messages for each and every step of the cross border payment flow, including reference data fields besides core XML data | 
+| 3. | ISO20022 XML <> Polkadot XCM data mapping | Identify data types gaps, XML<>XCM trnasformation and validations for the use case| 
+| 4. | ISO20022 <> Polkadot messaging | Identify XCMP gaps in messaging, relaying, routing feasibilities for the use case | 
+| 5. | Define offchain data storage | Define offchain data structure, storage and indexes for ISO20022 messages and payments in this use case |
+| 6. | Define offchian worker module | Define OCW logic to handle ISO20022 messages mapping, processing, messaging in this use case |
+| 7. | Define Smart contracts | Define ink! smart contracts to implement X-border payments and offchian integrations in this use case |
+| 8. | Define ISO20022 messaging and payment events and stats | Define ISO20020 messaging and payment flow events and states |
+| 9. | Define ISO20022 messaging and payment queries | Define how to query ISO20022 messaging and payments, based off events, stats, transactions and indexes stored offchain |
+
+
+### Milestone 2 Example — Additional features
```

Comment:
Please replace template text

Reply:
Replaced template text in this [commit](https://github.com/OpenSmartContract/W3F-Program/commit/16385bba4c45b5ba0dcd246542d4ba586e5e93c6)